pmm-app/src/pmm-dbaas/DBaaS.styles.ts

```diff
@@ -0,0 +1,8 @@
+import { CSSProperties } from 'react';
+
+export const styles = {
```

Comment:
We don't need css here because this is being used in "styles" attribute and not in "className". If we eventually need to add classes for sure we can use `css`. Furthermore className is not being used because grafana Tab component ignores it.

Reply:
True, the Tab component ignores `className` :( One more reason to have our own library.